src/client/js/otp/config.js

```diff
@@ -23,8 +23,8 @@ otp.config = {
         'sl': otp.locale.Slovenian,
         'fr': otp.locale.French,
         'it': otp.locale.Italian,
-        'ca_ES': otp.locale.Catalan,
-		'pt': otp.locale.Portuguese
+        'es': otp.locale.Spanish,
```

Comment:
Ops, it has been by mistake when mergin PT trasnalation... I'm fixing it right now